src/ep.config.json

```diff
@@ -1,8 +1,8 @@
 {
   "cortexApi": {
-    "path": "http://localhost:8080/cortex",
+    "path": "/cortex",
     "scope": "vestri",
-    "pathForProxy": "http://localhost:9080"
+    "pathForProxy": "http://vestri.epdemos.com"
```

Comment:
as Shaun pointed out, I don't think these values make sense OOB, these are more valid for running in Prod Mode, OOB I think we expect to be running in Dev Mode.

Reply:
I think we should keep the `cortexApi` to `/cortex`. I can revert this one if you think that `localhost` makes more sense for development. Although I would prefer not to have to run local docker instances anymore.